packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -187,6 +187,7 @@ const AccountSwitcher = props => {
     // mt_financial_company: { financial: {}, financial_stp: {}, swap_free: {} }
     // mt_gaming_company: { financial: {}, swap_free: {} }
     const getRemainingAccounts = (existing_cfd_accounts, platform) => {
+        const has_financial_account = existing_cfd_accounts.map(accounts => accounts.market_type).includes('financial');
```

Comment:
This logic already exists in `getCFDConfig` function. Please fix the issue in `getCFDConfig` function.

Reply:
Moved funtionality now to getCFDConfig